windows/cross-compile/check-dlls

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+
+ifeq ($(ARCH),i686)
+WINE_DLL_DIR="/usr/lib/x86_64-linux-gnu/wine/i386-windows"
```

Comment:
Should be `/usr/lib/i386-linux-gnu/wine/i386-windows`.

Reply:
Done.
<!-- Reviewable comment -ODrYXIC5nlWSMlCcrFw:b-896fix -->
<!-- Sent from Reviewable.io -->